src/components/EditSections/EditExtendedInfo/EditExtendedInfo.js

```diff
@@ -30,6 +30,10 @@ class EditExtendedInfo extends Component {
     onToggle: PropTypes.func.isRequired,
   };
 
+  state = { isFieldRequired: false };
+
+  toggleFieldRequired = (_, value) => this.setState({ isFieldRequired: !!value });
```

Comment:
Please, clarify the fields that will be marked as required in the function name.